src/components/SingleDatePickerInput.jsx

```diff
@@ -88,8 +88,6 @@ const defaultProps = {
   phrases: SingleDatePickerInputPhrases,
 };
 
-/* eslint react/no-this-in-sfc: 1 */
```

Comment:
lol thank you

Reply:
Lol, I think this was just... left over from a refactor. :O